examples/tools/meson/mesontoolchain/build_simple_meson_project.rst

```diff
@@ -0,0 +1,140 @@
+.. _examples_tools_meson_toolchain_build_simple_meson_project:
+
+Build a simple Meson project using Conan
+========================================
+
+In this example, we are going to create a string compressor application
+that uses one of the most popular C++ libraries: `Zlib <https://zlib.net/>`__.
+
+.. note::
+
+    This example is based on the main :ref:`Build a simple CMake project using Conan<consuming_packages_build_simple_cmake_project>`
+    tutorial. So we highly recommend reading it before trying out this one.
```

Comment:
Added ref to the main tutorial so it's not necessary to tell everything about Conan basics in this example